app/models/canonical_armor.rb

```diff
@@ -15,8 +29,8 @@ class CanonicalArmor < ApplicationRecord
   validates :body_slot,
             presence:  true,
             inclusion: {
-                         in:      %w[head body hands feet shield],
-                         message: 'must be "head", "body", "hands", "feet", or "shield"',
+                         in:      %w[head body hands feet hair shield],
```

Comment:
This turns out to be a relevant body slot for Skyrim, and it applies only to one or two specific items.